anypoint-mq/3.x/modules/ROOT/pages/anypoint-mq-listener.adoc

```diff
@@ -27,7 +27,11 @@ When a consumer takes a message from the queue and stores it in the local buffer
 +
 The smaller buffer restricts app-level throughput and avoids blocking competing consumers.
 
-When using the `prefetch` mode, the `acknowledgementTimeout` timer begins from the moment the message is dispatched to the flow, not when it is taken from the queue. The time the message remains in the local buffer doesn't count as part of the timeout.
+When using the `prefetch` mode, the `acknowledgementTimeout` timer begins when the message is taken from the queue. 
+Anypoint MQ attempts to honor the expiration time specified by `acknowledgementTimeout`.
+However, Anypoint MQ extends the expiration timeout if the time that the message remains in the buffer reaches 80% of the timeout value.
+
+Don't use `prefetch` mode for applications with long processing times or that require messages to be processed only once.
```

Comment:
I would remove the part of `that require messages to be processed only once.` I know it's specific to what this customer saw but its too generic and in most cases this is not true. 
I would just leave the first part and probably do:
`It is not recommended to use prefetch mode for applications with long processing times and max concurrency limitation since message will remain in buffer for some time` -> change to something nicer jajaj. I am not good at writing this stuff